flowmachine/flowmachine/features/location/redacted_unique_subscriber_counts.py

```diff
@@ -0,0 +1,47 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import List
+
+from flowmachine.core import Query
+from flowmachine.core.mixins import GeoDataMixin
+from flowmachine.features import UniqueSubscriberCounts
+from flowmachine.features.location.spatial_aggregate import SpatialAggregate
+
+
+class RedactedUniqueSubscriberCounts(GeoDataMixin, Query):
```

Comment:
The exposed "unique subscriber counts" is still just using `UniqueSubscriberCounts`, not `RedactedUniqueSubscriberCounts`.